client/pages/teams.jsx

```diff
@@ -0,0 +1,279 @@
+import React from 'react';
+import AppContext from '../lib/app-context';
+
+export default class Teams extends React.Component {
+
+  renderGroupA() {
+    const { teams } = this.context;
+    const groupA = [];
+    for (let i = 0; i < teams.length; i++) {
+      if (teams[i].group === 'A') {
+        groupA.push(teams[i]);
+      }
+    }
+    const groupRender = groupA.map(groupList => {
+      return (
+        <a className='team-link' key={groupList.teamId} href={`#teams/teamid=${groupList.teamId}`}>
+          <div className='team-wrapper my-2 mx-auto d-flex justify-content-start'>
+            <img className='team-flag me-4' src={groupList.countryFlag} alt="" />
+            <h1 className='team-name'>{groupList.countryName}</h1>
+          </div>
+        </a>
+      );
+    });
+
+    return (
+      <div className='d-flex flex-column'>
+        <a className='team-title-link mx-2' href="">
+          <div className='group-wrapper d-flex justify-content-center'>
+            <h2 className='group-title'>group a</h2>
+          </div>
+        </a>
+        <div className='mx-2'>{groupRender}</div>
+      </div>
+    );
+  }
```

Comment:
You could make a single method that does this, then pass the different group arrays to that method.